reactfire/rollup.config.js

```diff
@@ -1,21 +1,35 @@
-import resolve from 'rollup-plugin-node-resolve';
+import typescript from '@rollup/plugin-typescript';
 
-export default {
-  input: './pub/reactfire/index.js',
-  output: {
-    dir: './pub/reactfire/cjs',
-    format: 'cjs',
-    name: 'reactfire'
-  },
+const config = {
+  input: './index.ts',
+  plugins: [typescript()],
   external: [
-    'react',
     'firebase/app',
+    'react',
     'rxfire/auth',
     'rxfire/database',
     'rxfire/firestore',
     'rxfire/storage',
     'rxjs',
     'rxjs/operators'
-  ],
-  plugins: [resolve()]
+  ]
 };
+
+export default [
+  {
+    ...config,
+    output: {
+      dir: './pub/reactfire/cjs',
```

Comment:
What's the `reactfire` subdirectory for? I've seen others do it, but I don't understand why.

Reply:
I see now that this isn't something you added.